docs/scos/dev/front-end-development/202307.0/oryx/getting-started/set-up-oryx.md

```diff
@@ -16,21 +16,29 @@ This document describes how to set up an environment for developing in the Oryx
 
 ## Install Oryx with the boilerplate project
 
-1. Download the boilerplate project and install dependencies:
-
-```shell
-git clone https://github.com/spryker/composable-frontend && \
-cd composable-frontend && \
-npm i
-```
+1. Download the boilerplate project and install dependencies. There are two options:
+    - Default Oryx setup, which is most suitable for a general eCommerce web-application, like B2C or B2B shop:
+        ```shell
+        git clone https://github.com/spryker/composable-frontend && \
+        cd composable-frontend && \
+        npm i
+        ```
+
+    - Fulfillment application based on Oryx, which is a specific PWA setup perfectly fits Fulfillment flow with a few additional features, like offline mode, mobile-oriented UI, etc.:
+        ```shell
+        git clone https://github.com/spryker/composable-frontend && \
+        cd composable-frontend && \
+        git checkout fulfillment && \
+        npm i
+        ```
 
 2. Run the application:
 
-```shell
-npm run dev
-```
+    ```shell
+    npm run dev
+    ```
 
-The application gets available at `localhost:3000`.
+The application gets available at `localhost:3000` (or localhost:4200 for Fulfillment app).
```

Comment:
Mmm, shall we move FA to 3000 as well? No point in differentiating here imo. 4200 sounds like a default angular port, huh?

Reply:
Agree. For monorepo it make sense so you can easly work with FA and SF at the same time. But for boilerplate it make sense to use 3000 for both.